diffsync/enum.py

```diff
@@ -35,6 +35,29 @@ class DiffSyncModelFlags(enum.Flag):
     Can be used for the case where deletion of a model results in the automatic deletion of all its children.
     """
 
+    SKIP_UNMATCHED_SRC = 0b100
+    """Ignore the model if it only exists in the source/"from" DiffSync when determining diffs and syncing.
+
+    If this flag is set, no new model will be created in the target/"to" DiffSync.
+    """
+
+    SKIP_UNMATCHED_DST = 0b1000
+    """Ignore the model if it only exists in the target/"to" DiffSync when determining diffs and syncing.
+
+    If this flag is set, the model will not be deleted from the target/"to" DiffSync.
+    """
+
+    SKIP_UNMATCHED_BOTH = SKIP_UNMATCHED_SRC | SKIP_UNMATCHED_DST
+
+    CRUD_NO_UPDATE = 0b10000
```

Comment:
Shouldn't we have a CRUD_NO_CREATE flag (to be set on the "source" side) as well? 

Relatedly, should we allow for CRUD_NO_UPDATE to be set on either the "source" or "target" model instances, and prevent changes in either case?

Reply:
I thought about it too but I'm not sure if this is possible to have flags defined on the source side currently because we are only passing the dest model to the `sync_model` method.

I think it would require us to retrieve the `src_model` from the `src_diffsync` and pass it to `sync_model()`. I can give it a try but if it turns out to be too complicated I might change my mind

Reply:
I've made the changes mentioned previously, I think it's cleaner overall